src/test/robot/api/storage/pravega.robot

```diff
@@ -122,7 +122,7 @@ Events Pipeline Test
     ...  --storage-net-node-addrs=${node_addr}
     ...  --storage-namespace=scope7 \
     ...  --storage-driver-limit-concurrency=10
-    ...  --storage-driver-threads=10
+    ...  --storage-driver-threads=1
```

Comment:
what is the purpose of this change?

Reply:
it's written in the commit name. I was checking if the problem with e2e latency test fail is correlated with the empty reader we get from a pool. This change proved that it is indeed correlated.
As it was said in the commit- it's only a check